docs/github-setup.md

```diff
@@ -0,0 +1,27 @@
+# GitHub Setup
+
+## GitHub Organization
+
+We recommend that each hackweek creates a [GitHub Organization](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations ) to manage various repositories for an event. This makes organizing teams of users easy and consolidates content such as project work and tutorials for easier discoverability. It also allows managing secrets and making them available to different repositories if necessary.
+
+## Hackweek Bots
+
+For NASA-focused hackweeks we created a “bot” user with a gmail account, GitHub Account https://github.com/hackweek-admin, and NASA Earthdata login (https://urs.earthdata.nasa.gov ) to allow GitHub Actions workflows to access NASA Data with a set of credentials that do not belong to any individual.
+
+## GitHub Secrets
+
+We can add credentials as [“Organization Secrets”]https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions ) and make them available to only select repositories that need them. For example see the screenshots below on giving the `EARTHDATA_PASS` secret to a specific repository
+
+1. Start by going to the `Settings` and `Secrets` for a specific repository the URL will look like this https://github.com/snowex-hackweek/website-2024/settings/secrets/actions  
+
+**only people with `admin` access to the repository can access these settings**
+
+![Repo secrets](./images/github-repo-secrets.png)
+
+1. Select 'Manage organization secrets'. You can either create a new one, or click the "pencil" icon of an existing one to give access to a new repository
+
+![Org Secrets](./images/github-org-secrets-list.png)
+
+1. In the end, a website repository needs to have access to both `NETLIFY_AUTH_TOKEN` and a specific `NETLIFY_SITE_ID` for the preview workflow to function:
```

Comment:
Since we don't render this in the book, should we explicitly number this since GH doesn't fully render MD?